source/manual/fix-problems-with-vagrant.html.md

```diff
@@ -1,10 +1,10 @@
 ---
```

Comment:
Adding a redirect for this page would be good: https://github.com/alphagov/govuk-developer-docs/blob/master/data/redirects.yml